ext/afform/core/Civi/Api4/Action/CustomGroup/GetAfforms.php

```diff
@@ -285,17 +286,19 @@ private function generateTabForm($item): array {
       'permission' => ['access all custom data'],
       'title' => $item['title'],
       'icon' => $item['icon'],
+      'summary_weight' => 100 + ($item['weight'] ?? 0),
     ];
     $entityIdFilter = \CRM_Utils_String::convertStringToSnakeCase($item['extends']) . '_id';
-    // Place in Contact Summary Tabs
-    if ($item['extends'] === 'Contact') {
-      $afform['placement'] = ['contact_summary_tab'];
-    }
-    elseif (CoreUtil::isContact($item['extends'])) {
+    if (CoreUtil::isContact($item['extends'])) {
       // override e.g. "individual_id", we want "contact_id"
       $entityIdFilter = 'contact_id';
       $afform['placement'] = ['contact_summary_tab'];
-      $afform['summary_contact_type'] = [$item['extends']];
+      if (!empty($item['extends_entity_column_value'])) {
+        $afform['summary_contact_type'] = (array) $item['extends_entity_column_value'];
+      }
+      elseif ($item['extends'] !== 'Contact') {
```

Comment:
```suggestion
      if ($item['extends'] !== 'Contact') {
```

Probably unnecessary but won't test tonight - just a thought. https://github.com/civicrm/civicrm-core/pull/32082#discussion_r1955342138

Reply:
@ufundo no we want it the way it is because if there's a contact subtype we *don't* want the contact type to override it.

Reply:
oh sorry right they use the same slot in the afform meta... i see!

i hope you cant have 2 different types of VIP then..

Reply:
No, there's a unique index on `name`
https://github.com/civicrm/civicrm-core/blob/7befa582c4d1ad545616ef2990349656bd441e42/schema/Contact/ContactType.entityType.php#L20-L25